backend/src/supabase/functions/room-jwtprovider/index.ts

```diff
@@ -15,13 +17,16 @@ async function generateToken(supabaseClient, spaceId) {
       status: 404
     });
   }
-  let baseOptions = {
-    keyId: Deno.env.get('SPACE_KEY_ID'),
-    keySecret: Deno.env.get('SPACE_PRIVATE_KEY'),
-  };
-  const spaceToken = Mux.JWT.signSpaceId(spaceId, {
-    ...baseOptions,
-  });
+  
+  const expiration = new Date().valueOf() + 60*60;
```

Comment:
How long do we want the JWT to be valid for? As I understand, this line makes it valid for 3600 ms: Is this enough?

Reply:
You are right, it doesn´t seem to be enough. Mux's SDK defualts to 7 days... Maybe we can pass the expiration as a parameter and default it to 7 days, until we add the logic to decide how long a room should be kept alive. What do you think?

Reply:
So this is just the validity of the JWT, so we could let it expire sooner even if the room will be kept alive for long. We sign a new JWT every time we join a room (I think?), so it shouldn't be a big deal, I believe.

Does Mux use the default if we don't specify an expiration? We could either use the default, or set the validity for a few hours, just enough for an event.

Reply:
You're right, each time you enter a room you get a new JWT so it shouldn't be a big deal, but maybe adding a bit more validity makes sense. 

Mux's SDK gives a default expiration but they wrap the JWT signer, JWT's require an expiration. I suggest:

- Setting the default duration to 6 hours (should be more than enough for an event)
- Add a parameter to set custom expiration

Reply:
I agree, sounds good!